agent/engine/docker_container_engine.go

```diff
@@ -91,6 +91,7 @@ type DockerClient interface {
 	Version() (string, error)
 	InspectImage(string) (*docker.Image, error)
 	RemoveImage(string, time.Duration) error
+	LoadImage(opts docker.LoadImageOptions) error
```

Comment:
If this is just for testing, we could use either but go-dockerclient doesn't expect a pointer to LoadImageOptions